### PR TITLE
TSK-493 Check classification Id is null on creation

### PR DESCRIPTION
**Pull request by MiguelMartinRojas**
- Check classification id is null on classification creation.

### CODE REVIEW

File: rest/taskana-rest-spring-example/src/test/java/pro/taskana/rest/ClassificationControllerIntTest.java [hunk 1]
```diff
@@ -126,7 +130,7 @@ public void testGetSecondPageSortedByKey() {
 
     @Test
     public void testCreateClassification() throws IOException {
-        String newClassification = "{\"classificationId\":\"\",\"category\":\"MANUAL\",\"domain\":\"DOMAIN_A\",\"key\":\"NEW_CLASS\",\"name\":\"new classification\",\"type\":\"TASK\"}";
+        String newClassification = "{\"classificationId\":null,\"category\":\"MANUAL\",\"domain\":\"DOMAIN_A\",\"key\":\"NEW_CLASS\",\"name\":\"new classification\",\"type\":\"TASK\"}";
```
Comment (holgerhagen) [inline rest/taskana-rest-spring-example/src/test/java/pro/taskana/rest/ClassificationControllerIntTest.java]: Empty classification id is also valid, isn't it? Why did you change the code here?

Comment (holgerhagen) [inline rest/taskana-rest-spring-example/src/test/java/pro/taskana/rest/ClassificationControllerIntTest.java]: We should also have a test returning a HTTP 400 if ID is set.

File: lib/taskana-core/src/test/java/pro/taskana/impl/ClassificationServiceImplTest.java [hunk 2]
```diff
@@ -290,19 +291,38 @@ public void testGetClassificationWithInvalidNullKey() throws ClassificationNotFo
         }
     }
 
+    @Test(expected = InvalidArgumentException.class)
+    public void testGetClassificationWithInvalidNullId()
```
Comment (holgerhagen) [inline lib/taskana-core/src/test/java/pro/taskana/impl/ClassificationServiceImplTest.java]: this should probably be named like "testThrowExceptionIdClassificationIsCreatedWithAnExplicitId" or similar. What do you think?

Comment (MiguelMartinRojas) [reply lib/taskana-core/src/test/java/pro/taskana/impl/ClassificationServiceImplTest.java]: Yes, you are right your name is more descriptive.

File: rest/taskana-rest-spring-example/src/test/java/pro/taskana/rest/ClassificationControllerIntTest.java (hunk 1 above)
Comment (MiguelMartinRojas) [reply rest/taskana-rest-spring-example/src/test/java/pro/taskana/rest/ClassificationControllerIntTest.java]: ok
